docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/04_Images_And_Documents.md

```diff
@@ -70,6 +70,12 @@ Basic functionality and options for files:
 
 ![Inserted image](../../_images/inserted-image.png)
 
+## File usage
+
+The Used on tab allows a CMS content author to view the usage of a file across the CMS. Click the ***Used on*** tab in the File details panel. The first row shows this file is being used on a [Content block](/optional_features/content_blocks/) and also provides the Page name of where the Content block has been added in the CMS. To navigate to where a particular file is being used, click on the relevant link.
```

Comment:
A link is blue, please replace with row. As you hover over the whole gridfield row.

Reply:
I just mentioned previously https://github.com/silverstripe/silverstripe-userhelp-content/pull/154#discussion_r508179414 why I changes this from 'row' to 'link'.

Does that make sense?  We could change it to 'relevant link within the row' if you think that's better?

Reply:
I'd need to do a peer design review first before I confirm wording. Leave for now and we can change when other work has been completed.